libs/wire-api-federation/src/Wire/API/Federation/API.hs

```diff
@@ -71,14 +72,19 @@ type HasUnsafeFedEndpoint comp api name = 'Just api ~ LookupEndpoint (FedApi com
 
 -- | Return a client for a named endpoint.
 --
--- This function introduces an 'AddAnnotation' constraint, which is
+-- this function introduces an 'addannotation' constraint, which is
```

Comment:
Have you accidentally changed the cases in this line?

Reply:
I guess I did. Fixed now.